### ISSUE TITLE
tox uses wrong interpreters for obsolete Python versions

### ISSUE THREAD

**Issue by funkyfuture** [tox-dev/tox#1168]
ciao ragazzi, here's a

##### bug description

`tox` does not install some CPython versions, but rather uses the version of the interpreter that runs `tox`. this can be inspected with the logs of the builds in the "Allowed Failures" section of [this build](https://travis-ci.org/pyeve/cerberus/builds/491023009) on Travis. i could reproduce the behaviour locally with 2.4, 2.6 and 3.3 (i checked no further), e.g.:

```
$ TOXENV=py24 tox

GLOB sdist-make: /home/user/cerberus/setup.py
py24 recreate: /home/user/cerberus/.tox/py24
py24 installdeps: pytest
py24 inst: /home/user/cerberus/.tox/.tmp/package/1/Cerberus-1.2.zip
py24 installed: atomicwrites==1.3.0,attrs==18.2.0,Cerberus==1.2,more-itertools==6.0.0,pluggy==0.8.1,py==1.7.0,pytest==4.3.0,six==1.12.0
py24 run-test-pre: PYTHONHASHSEED='3823316362'
py24 runtests: commands[0] | pytest cerberus/tests
============================= test session starts ==============================
platform linux -- Python 3.6.3, pytest-4.3.0, py-1.7.0, pluggy-0.8.1
cachedir: .tox/py24/.pytest_cache
rootdir: /home/user/cerberus, inifile: pytest.ini
```

and some

##### environment information

```
$ cat /etc/issue
Ubuntu 18.10 \n \l

$ python2.4 -c "import sys; print sys.version_info"
(2, 4, 6, 'final', 0)

$ source ~/.local/venvs/tox/bin/activate  # tox is managed w/ pipsi

$ pip list
Package    Version
---------- -------
filelock   3.0.10 
pip        18.1   
pluggy     0.8.0  
py         1.7.0  
setuptools 40.5.0 
six        1.11.0 
toml       0.10.0 
tox        3.7.0  
virtualenv 16.1.0 
wheel      0.30.0 
You are using pip version 18.1, however version 19.0.3 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
```

**Comment by gaborbernat** [comment]
Agreed we should still try to create the correct one (though no guarantees offered that creation will pass). Do post a tox.ini that reproduces this.

**Comment by funkyfuture** [comment]
> Agreed we should still try to create the correct one

i don't understand. are you saying that `tox` ignores these values by default?

the used `tox.ini` [is here](https://github.com/pyeve/cerberus/blob/7e82a6aad3760451eedd647c2eb177f65add5d47/tox.ini).

**Comment by asottile** [comment]
Bisected this, #983 caused this regression

**Comment by asottile** [comment]
fixed again in #1377
